Concepts/Classes/Threading/Threading.cs

```diff
@@ -151,9 +151,7 @@ public void RaiseAbort(System.Threading.Thread thread)
 
             thread.Priority = (System.Threading.ThreadPriority)AbortPriority;
 
-            try { thread.Abort(); }
```

Comment:
I left these in place just in case Abort was ever restored... I don't have that big of objections about changing this as long as its consistent.